### PR TITLE
[Merged by Bors] - feat(Circle): More basic lemmas

### PR DESCRIPTION
**Pull request by YaelDillies**
From LeanAPAP

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/SpecialFunctions/Complex/Circle.lean [hunk 1]
```diff
@@ -73,6 +73,16 @@ lemma periodic_exp : Periodic exp (2 * π) := fun z ↦ exp_eq_exp.2 ⟨1, by rw
 
 @[simp] lemma exp_two_pi : exp (2 * π) = 1 := periodic_exp.eq.trans exp_zero
 
+lemma exp_int_mul_two_pi (n : ℤ) : exp (n * (2 * π)) = 1 :=
+  ext $ by simpa [mul_assoc] using Complex.exp_int_mul_two_pi_mul_I n
```
Comment (loefflerd) [inline Mathlib/Analysis/SpecialFunctions/Complex/Circle.lean:77]: Isn't `$` deprecated in mathlib?

Comment (YaelDillies) [reply Mathlib/Analysis/SpecialFunctions/Complex/Circle.lean:77]:
```suggestion
  ext <| by simpa [mul_assoc] using Complex.exp_int_mul_two_pi_mul_I n
```

**Comment by loefflerd** [comment]
LGTM

maintainer merge

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**